src/main/java/com/vdurmont/emoji/EmojiParser.java

```diff
@@ -99,6 +99,24 @@ public String transform(UnicodeCandidate unicodeCandidate) {
    * their unicode.
    */
   public static String parseToUnicode(String input) {
+      return parseToUnicode(input, false);
+  }
+
+  /**
+   * Replaces the emoji's aliases (between 2 ':') occurrences and the html
+   * representations by their unicode.<br>
+   * Examples:<br>
+   * <code>:smile:</code> will be replaced by <code>😄</code><br>
+   * <code>&amp;#128516;</code> will be replaced by <code>😄</code><br>
+   * <code>:boy|type_6:</code> will be replaced by <code>👦🏿</code>
+   *
+   * @param input the string to parse
+   * @param parseAsText force the client to render emojis as text
+   *
+   * @return the string with the aliases and html representations replaced by
+   * their unicode.
+   */
+  public static String parseToUnicode(String input, Boolean parseAsText) {
```

Comment:
What would you think of adding a new method (`parseToUnicodeAsText`) instead of modifying the API of this one?